.travis.yml

```diff
@@ -15,6 +15,8 @@ script:
   - npm run test
 before_deploy:
   - pip install --user awscli
+  - npm install -g file-size-report
```

Comment:
Would it not be better to add `file-size-report` as a `devDependency` of the root `package.json`?